### PR TITLE
Fix python 3 error message issue

### PR DESCRIPTION
**Pull request by julianandrews**
e.message isn't defined in python3

### CODE REVIEW

File: dinero/gateways/authorizenet_gateway.py [hunk 1]
```diff
@@ -553,7 +553,9 @@ def create_customer(self, options):
             if error_code == 'E00039':  # Duplicate Record
                 e.customer_id = None
 
-                customer_match = re.search(r'^A duplicate record with ID (.*) already exists.$', e.message[0][1])
+                customer_match = re.search(
+                    r'A duplicate record with ID (.*) already exists.', str(e)
+                )
```
Comment (acatton) [inline dinero/gateways/authorizenet_gateway.py]: I don't understand, but I could me mistaking. `e.message` is suppose to be a list of all the collected error messages. It looks like you're trying to find a string in the repr of the dict? That's not very optimal. How is that related to python 3?

Comment (julianandrews) [inline dinero/gateways/authorizenet_gateway.py]: Well, the issue is that in python3 `e.message` doesn't exist, so the options are either to access `e.args` or `str(e)`. I couldn't see a way to use `e.args` that would work in both python2 and python3, so the options are either to break it into cases, or just look at the `str`. I'm not super happy with this, but it seems like the simpler of the two options?

**Comment by julianandrews** [comment]
Possibly the tests should be moved over to some unittest framework, but I'm not doing that now!
